frontend/src/app/main/store.cljs

```diff
@@ -28,6 +29,28 @@
 (def on-event identity)
 
 (def ^:dynamic *debug-events* false)
+(def ^:dynamic *debug-events-time* false)
+
+(def current-measure (atom nil))
+
+(defn measure-time-to-render [event]
+  (if @current-measure
+    (swap! current-measure conj event)
```

Comment:
Maybe we need to delimit the number of elements. I know, it is a debug utility, but grow a lot i think